Hakawai/Mentions/HKWMentionsPlugin.m

```diff
@@ -1436,6 +1565,9 @@ - (void)dataReturnedWithEmptyResults:(BOOL)isEmptyResults
 }
 
 -(void) textViewDidProgrammaticallyUpdate:(UITextView *)textView {
+    if (HKWTextView.enableSimpleRefactor) {
+        return;
+    }
     if (self.state == HKWMentionsStartDetectionStateCreatingMention) {
```

Comment:
since we are ignoring those calls, what happens to those states? are we still reusing the same states, or simplifying the states or replacing with new sets of states?

Reply:
will fix this by creating v2 mentions plugin